examples/z_queryable_shm.c

```diff
@@ -31,7 +31,7 @@ void query_handler(const z_loaned_query_t *query, void *context) {
 
     const z_loaned_bytes_t *payload = z_query_payload(query);
     if (z_bytes_len(payload) > 0) {
-        z_loaned_shm_t *shm = NULL;
+        const z_loaned_shm_t *shm = NULL;
```

Comment:
This shouldn't be `const`, isn't it? We are deserializing into it.

Reply:
We are getting pointer's value, not deserializing into it (as it is pointing to NLL)